platforms/corda-ent/charts/float/templates/_helpers.tpl

```diff
@@ -0,0 +1,5 @@
+{{- define "labels.custom" }}
+  {{ range $key, $val := $.Values.metadata.labels }}
+  {{ $key }}: {{ $val }}
+  {{ end }}
+{{- end }}
```

Comment:
Add newline

Reply:
Fixed in [9862bdb](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/893/commits/9862bdbebd85ac9508aaec96a1807981550cf571)